README.md

```diff
@@ -63,6 +76,18 @@ To view a module from the configured modules database:
 
 The module will appear read-only in a new text buffer.
 
+
+### Running MarkLogic Unit Tests
+
+This plugin provides a convenient method for running [MarkLogic Unit Tests](https://marklogic-community.github.io/marklogic-unit-test/) within your MarkLogic server. To get started, your test suites and files must be organized under a "test/suites" directory. Additionally, you need to set the `Marklogic: Unit Test Port` setting to the port number of the App Server that can run your unit tests. Finally, to run a test file:
```

Comment:
Re: where the test modules should be - unfortunately, the marklogic-unit-test docs are not clear about this - https://marklogic-community.github.io/marklogic-unit-test/docs/conventions/ . In fact, the guidance of using "src/test/suites" is most likely not going to work. In practice, the correct path will usually be "src/test/ml-modules/root/test/suites". 

The only good advice from the docs is in the "Getting Started" page - https://marklogic-community.github.io/marklogic-unit-test/ - where it recommends looking at ml-gradle's example project, which does show the correct path being used. Sigh. 

Clearly the marklogic-unit-test docs need to be improved. Until then, I think these docs should go with what's going to work the majority of the time, which is to do the following:

1. State that the files should be under "src/test/ml-modules/root/test/suites".
2. Point the user to the ml-gradle example project for an example of how to setup the project to use marklogic-unit-test. 

Ideally, we'll improve the marklogic-unit-test docs soon to make it crystal clear how to setup your project, and then the mlxprs docs can simply refer to that.